PySpice/Spice/HSpice/RawFile.py

```
@@ -0,0 +1,250 @@
+import logging
+
+import numpy as np
+
+from PySpice.Probe.WaveForm import (AcAnalysis, DcAnalysis, OperatingPoint,
+                                    TransientAnalysis, WaveForm)
+from PySpice.Unit import u_A, u_Degree, u_Hz, u_s, u_V
+
+
+class AnalysisList(list):
+    def __init__(self, analyses, measurements=None):
```

Ruff (0.15.20)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)